app/views/gsa18f/procurements/form.html.haml

```diff
@@ -0,0 +1,25 @@
+- content_for :title, "GSA 18f Purchase Request"
```

Comment:
this form is BEAUTIFUL (esp compared to old form) ...are we generally moving everything to HAML in this project?